models/staging/stg_workday__job_family_group.sql

```diff
@@ -25,7 +25,7 @@ final as (
     
     select 
         source_relation, 
-        _fivetran_deleted,
+        _fivetran_deleted, 
```

Comment:
A large number of these staging models include the `_fivetran_deleted` field. Usually we will apply a filter to remove the deleted records in the staging models. Do you feel we should also apply that same update for Workday staging models?

Reply:
Agreed, updated